packages/terra-form/src/Input.jsx

```diff
@@ -55,7 +55,8 @@ class Input extends React.Component {
       ...customProps
     } = this.props;
     const additionalInputProps = Object.assign({}, customProps);
-    const inputClasses = cx(['input', additionalInputProps.className]);
+    const inputClasses = cx([(customProps.type === 'radio' || customProps.type === 'checkbox') ?
+      '' : 'input', additionalInputProps.className]);
```

Comment:
Put each class on a new line for readability and use first class conditional checks provided by classNames as Stephen mentioned in the Control.